st2api/st2api/cmd/api.py

```diff
@@ -74,8 +74,10 @@ def main():
         return _run_server()
     except SystemExit as exit_code:
         sys.exit(exit_code)
-    except Exception:
-        LOG.exception('(PID=%s) ST2 API quit due to exception.', os.getpid())
+    except Exception as e:
+        msg = getattr(e, 'message', str(e))
+        LOG.error('(PID=%s) ST2 API quit due to exception.', os.getpid())
```

Comment:
I actually decided to revert the logging change since all the services do it like that.

It will cause stack trace to be printed twice in some scenarios, but oh well.